pkg/resourcemanager/webhook/add.go

```diff
@@ -76,9 +76,11 @@ func AddToManager(mgr manager.Manager, sourceCluster, targetCluster cluster.Clus
 
 	if cfg.Webhooks.HighAvailabilityConfig.Enabled {
 		if err := (&highavailabilityconfig.Handler{
-			Logger:        mgr.GetLogger().WithName("webhook").WithName(highavailabilityconfig.HandlerName),
-			TargetClient:  targetCluster.GetClient(),
-			TargetVersion: targetVersion,
+			Logger:                              mgr.GetLogger().WithName("webhook").WithName(highavailabilityconfig.HandlerName),
+			TargetClient:                        targetCluster.GetClient(),
+			TargetVersion:                       targetVersion,
+			DefaultNotReadyTolerationSeconds:    cfg.Webhooks.HighAvailabilityConfig.DefaultNotReadyTolerationSeconds,
+			DefaultUnreachableTolerationSeconds: cfg.Webhooks.HighAvailabilityConfig.DefaultUnreachableTolerationSeconds,
```

Comment:
Instead of passing the values individually, what about passing the entire webhook config like we do it for controllers?